stacks/openshift/application/oidc-authservice-for-appid/params.env

```diff
@@ -0,0 +1,9 @@
+client_id=xxxxxx
+oidc_provider=https://us-east.appid.cloud.ibm.com/oauth/v4/xxxx
+oidc_redirect_uri=https://<kubeflow-gateway-host>/login/oidc
+oidc_auth_url=
+application_secret=yyyyyy
+skip_auth_uri=
+namespace=istio-system
+userid-header=kubeflow-userid
```

Comment:
userid-header and userid-prefix should come from the global kubeflow-config, right?

Reply:
Yes, again this is one piece of config in namespace `istio-system`, I'm happy to refactor it when enabling this new kustomize config in any kfdefs.